src/main/java/org/radarbase/ksql/udaf/UniformSamplingReservoirDoubleUdaf.java

```diff
@@ -0,0 +1,70 @@
+package org.radarbase.ksql.udaf;
+
+import io.confluent.ksql.function.udaf.Udaf;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+
+public abstract class UniformSamplingReservoirDoubleUdaf implements Udaf<Double, Struct, Double> {
+
+    public static final String SAMPLES = "SAMPLES";
+    public static final String COUNT = "COUNT";
+    public static final Schema STRUCT_RESERVOIR = SchemaBuilder.struct().optional()
+            .field(SAMPLES, SchemaBuilder.array(Schema.OPTIONAL_FLOAT64_SCHEMA).optional().build())
+            .field(COUNT, Schema.OPTIONAL_INT64_SCHEMA)
+            .build();
+    private static final int MAX_SIZE_DEFAULT = 9999;
```

Comment:
I'd think that 5000 or even 1000 would be enough? I'm not sure what precision the different measures need though. For mode, median and quartiles 1000 would be enough. Perhaps for skew or stddev more precision is needed?

Reply:
Yes, I will reduce this since we are also using Double now.